### DISCUSSION TITLE
Failed to build installable wheels for some pyproject.toml based projects (simpleaudio)

### DISCUSSION THREAD

**Discussion by xianhua2020** [lookatwallstreet/Program-Trading-Based-on-Webull#22] [Q&A] [answered]
I installed program-trading-based-on-webull on my rockylinux and see this error:
      In file included from c_src/posix_mutex.c:7:
      c_src/simpleaudio.h:10:10: fatal error: Python.h: No such file or directory
         10 | #include <Python.h>
            |          ^~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for simpleaudio
  Running setup.py clean for simpleaudio
Failed to build simpleaudio
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (simpleaudio)

Do you know how to fix it?

**Comment by LukeWang01** [comment, marked as answer]
it's the package issue: simpleaudio

please refer to:

https://github.com/LukeWang01/Program-Trading-Based-on-Webull/blob/main/docs/fix_install_error.md
